src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -44,7 +44,7 @@ class JesBackendCall(val backend: JesBackend,
   import JesBackend._
   import JesBackendCall._
 
-  def jesCommandLine = s"/bin/bash ${cmdInput.local}"
+  def jesCommandLine = s"/bin/bash ${JesWorkingDisk.MountPoint}/${cmdInput.local}"
```

Comment:
I know we do this everywhere but IMO it'd be better to use Paths.get(...) w/ `resolve`s as much as possible (or even `cromwell.engine.PathString.toPath` when possible), instead of hardcoding `/`